sonar-project.properties

```diff
@@ -1,6 +1,6 @@
 sonar.projectKey=fh-template-apps
 sonar.projectName=fh-template-apps-nightly-master
-sonar.projectVersion=5.0.19
+sonar.projectVersion=5.0.20
```

Comment:
Note that the version in the package.json didn't get bumped here since it was already on 5.0.20. This change probably warrants a bump, whatever that should be.

Reply:
:+1:

Reply:
+1 Sometimes people forget to bump the sonar properties. This should be 5.0.21 in both files. Other changes look good to me.